satellite/audit/verifier.go

```diff
@@ -698,3 +717,41 @@ func rebuildStripe(ctx context.Context, fec *infectious.FEC, corrected []infecti
 	}
 	return stripe, nil
 }
+
+// GetRandomStripe takes a pointer and returns a random stripe index within that pointer.
+func GetRandomStripe(ctx context.Context, pointer *pb.Pointer) (index int64, err error) {
+	defer mon.Task()(&ctx)(&err)
+	redundancy, err := eestream.NewRedundancyStrategyFromProto(pointer.GetRemote().GetRedundancy())
+	if err != nil {
+		return 0, err
+	}
+
+	// the last segment could be smaller than stripe size
+	if pointer.GetSegmentSize() < int64(redundancy.StripeSize()) {
+		return 0, nil
+	}
+
+	var src cryptoSource
+	rnd := rand.New(src)
```

Comment:
We technically don't need cryptographic randomness, having a good starting seed with really good crypto generator should be sufficient.

Reply:
I think it might have been @kaloyan-raev who originally added the cryptographic randomness? I'm going to just move it to a separate file for now I think.

Reply:
I'm also not totally sure what "having a good starting seed with a really good crypto generator" means in terms of what we should keep and get rid of.

Reply:
"A good starting seed with a really good crypto generator" is basically the definition of cryptographic randomness :)

I believe the randomness in this case does need to be crypto-strength, to make things harder for attackers who want to subvert the auditing. I certainly agree with moving it into a separate file, or maybe even a separate package under `internal/`?

Reply:
moving it to `internal` might be overkill.